published/012-2016-August-24.md

```diff
@@ -0,0 +1,110 @@
+
+![textui](orbit-textui)
+
+Orbit also now has a [textui](https://github.com/haadcode/orbit-textui). This uses [blessed](https://github.com/chjj/blessed), a curses-like library with a high level terminal interface API for Node.js. This means you can chat to people using Orbit, straight from where you get your work done, in the terminal. Of course, other venues, [like Slack](https://github.com/haadcode/orbit/issues/71), are also in the pipeline.
+
+If you're interested in getting involved with developing Orbit and helping us get there faster, we've been adding a lot more 'Help Wanted' labels to issues that we think could use some work. Look at [these issues](https://github.com/haadcode/orbit/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+sort%3Areactions-%2B1-desc) and dive in. If you have questions, just jump on [the IRC channel](https://webchat.freenode.net/?channels=%23ipfs), or comment on the issues themselves.
+
+### [go-ipfs](//github.com/)
+
+We updated `go-ipfs` to 0.4.3-rc3 on August 8th. As we state in the [changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md), this release candidate fixes a panic that occurs when input from stdin was expected, but none was given. Check out [this issue](https://github.com/ipfs/go-ipfs/pull/3050) for details. If you missed the 0.4.3 roll-out, you can take a look at that changelog [here]().
+
+Moving to go1.7 is in the works, as it will give us performance improvements, `context` has been added into stdlib, and `net.Diallib` will allow our calls to vastly improve resource consumption and downtime. It also adds support for macOS Sierra. [Track that here](https://github.com/ipfs/go-ipfs/issues/3093).
+
+We're working on getting code coverage for all of the packages included in go-ipfs. To follow along or comment, [go here](https://github.com/ipfs/go-ipfs/issues/3053). We also plan on using coveralls to have solid tests for subprojects; follow that [here](https://github.com/ipfs/go-ipfs/issues/3112).
+
+On a related note, @whyrusleeping also udpated `gx deps --tree`, and it's related options, which means that the [gx](https://github.com/whyrusleeping/gx/) package manager we use in IPFS is now easier to use.
+
+Moving forward, we have a few main goal: Shipping IPFS with IPLD, interop with js-ipfs, and reliability and resource consumption. @whyrusleeping has written a [Captain's Log](https://github.com/ipfs/go-ipfs/issues/2247#issuecomment-238675394) outlining his vision of what is happening and where to track changes. If you're interested in getting involved with developing go-ipfs, we've been adding a lot more 'Help Wanted' labels to issues (just like we have for Orbit). Look at [these issues](https://github.com/ipfs/go-ipfs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) and dive in. If you have questions, just jump on [the IRC channel](https://webchat.freenode.net/?channels=%23ipfs), or comment on the issues themselves.
```

Comment:
> we have a few main goal: Shipping IPFS with IPLD, interop with js-ipfs, and reliability and resource consumption

goals plural, and resource consumption is not a goal, but the reduction thereof :)